jobs/python.setup.py.yml

```diff
@@ -15,53 +15,34 @@ parameters:
   # Name of 'Service connection' defined in Azure DevOps project settings
   # See https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/package/twine-authenticate?view=azure-devops#arguments
   pypiConnector: "testpypi-tomtom-dev"
-  pythonUseVersion: "3.6"
+  pythonDeployVersion: "3.6"
+  pythonTestVersions:
+  - name: Python35
+    version: "3.5"
+  - name: Python36
+    version: "3.6"
+  - name: Python37
+    version: "3.7"
+  testJobs:
```

Comment:
I would name this just `jobs`

Reply:
I'm ok making this change, since it is shorter and testJobs isn't quite right, but it does have the downside of not encompassing all jobs, but rather just the pre-deploy checks.